src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -126,6 +126,13 @@ class ElasticIsotropicUpdates : public SolidBaseUpdates
                                  localIndex const q,
                                  real64 ( &elasticStrain )[6] ) const override final;
 
+  GEOSX_HOST_DEVICE
+  virtual real64 getBulkModulus( localIndex const k ) const override final
```

Comment:
Just `final`.

Reply:
should I remove `override`?

Reply:
Hmm, maybe not. You may need `override` indeed.

Reply:
https://stackoverflow.com/questions/29412412/does-final-imply-override